src/desktop/apps/auction_support/test/client/registration_form.test.coffee

```diff
@@ -89,11 +89,8 @@ describe 'RegistrationForm', ->
         html = @view.$el.html()
         # FIXME: name error not rendered, bluebird hides failure
```

Comment:
We can probably remove this `FIXME` and the line below since they're not longer relevant.